packages/xstate-vue/test/UseService.vue

```diff
@@ -14,20 +15,24 @@ import {
   spawn,
   doneInvoke,
   State,
-  Service
 } from 'xstate';
-import { watch, ref } from '@vue/composition-api';
+import { watchEffect, ref, Ref } from '@vue/composition-api';
```

Comment:
both `ref` and `Ref` are unused in this file, could you remove them?

Reply:
yea found these test .vue files are hard to type/lint. Will clean them up a bit.